src/main/webapp/app/exercises/programming/shared/code-editor/layout/code-editor-grid.component.html

```diff
@@ -10,7 +10,10 @@
             <div class="editor-sidebar-left collapsable">
                 <ng-content select="[editorSidebarLeft]"></ng-content>
                 <!-- Required for resizing; don't remove empty span -->
-                <fa-icon [icon]="'grip-lines-vertical'" class="rg-sidebar-left text-lightgrey"><span></span></fa-icon>
+                <fa-icon *ngIf="!fileBrowserIsCollapsed" [icon]="'grip-lines-vertical'" id="draggableIconForFileBrowser" class="rg-sidebar-left text-lightgrey">
+                    <span></span>
```

Comment:
only for my understanding - why is the empty span needed here? It also appears in the other icons in this component.

Reply:
I think this is needed for the correct placement/fontsize of the icon for some reason, but I honestly can't tell you why. I guess we could check if it still works without it